webhook_server/tests/conftest.py

```
@@ -200,3 +203,123 @@ def owners_files_test_data():
+        """
+        log_file = log_dir / filename
+        with open(log_file, "w", encoding="utf-8") as f:
+            for entry in entries:
+                f.write(json.dumps(entry) + "\n")
+        return log_file
+
+    return _create_json_log_file
+
+
+@pytest.fixture
+def create_text_log_file(tmp_path):
```

Ruff (0.14.13)

Unused function argument: `tmp_path`

(ARG001)